Sources/Winget-AutoUpdate/Winget-Install.ps1

```diff
@@ -312,6 +312,7 @@ $Script:IsElevated = $CurrentPrincipal.IsInRole([Security.Principal.WindowsBuilt
 #Get WAU Installed location
 $WAURegKey = "HKLM:\SOFTWARE\Romanitho\Winget-AutoUpdate\"
 $Script:WAUInstallLocation = Get-ItemProperty $WAURegKey -ErrorAction SilentlyContinue | Select-Object -ExpandProperty InstallLocation
+$Script:WAUModsLocation = $WAUInstallLocation + "mods"
```

Comment:
If I could make a suggestion:
`$Script:WAUModsLocation = Join-Path -Path $WAUInstallLocation -ChildPath "mods"`
Just in case the `'/'` disappears/reappears.